backend/coins/btc/exchange.go

```diff
@@ -32,10 +32,10 @@ func (account *Account) SafelloBuySupported() bool {
 func (account *Account) SafelloBuy() *safello.Buy {
 	switch account.coin.Net().Net {
 	case chaincfg.MainNetParams.Net:
-		address := account.GetUnusedReceiveAddresses()[0]
+		address := account.GetUnusedReceiveAddresses()[0].Addresses[0]
```

Comment:
Why are we still keeping Safello?

Reply:
they promised to restore service by Q1 2020, so we made an issue to wait a bit and remove it if that didn't happen. obviously it didn't; i'll reach out quickly to see if there is any hope left.